### PR TITLE
Update draft-ietf-rats-ar4si.md

### PR DESCRIPTION
**Pull request by ericvoit**

### CODE REVIEW

File: draft-ietf-rats-ar4si.md [hunk 1]
```diff
@@ -197,13 +197,13 @@ The following sections detail requirements for these three categories.
...
-Attester identities may then be acquired through signed communications with the Verifier identity and/or the pre-provisioning Attester public keys in the Attester.
+Attester identities may then be acquired through signed or encrypted communications with the Verifier identity and/or the pre-provisioning Attester public keys in the Attester.
 
-During the Remote Attestation process, the Verifier's identity will be established with a Relying Party via a Verifier signature across recent Attestation Results.
+During the Remote Attestation process, the Verifier's identity must be established with a Relying Party, often via a Verifier signature across recent Attestation Results.
 This Verifier identity could only have come from a key pair maintained by a trusted developer or operator of the Verifier.
 
 Additionally, each set of Attestation Results must be provably and non-reputably bound to the identity of the original Attesting Environment which was evaluated by the Verifier.
-This will be accomplished via two items.
+This will be accomplished via two the meeting of two requirements.
```
Comment (henkbirkholz) [inline draft-ietf-rats-ar4si.md:206]:
```suggestion
This is accomplished via satisfying two requirements.
```

File: draft-ietf-rats-ar4si.md [hunk 2]
```diff
@@ -197,13 +197,13 @@ The following sections detail requirements for these three categories.
 Identity Evidence must be conveyed during the establishment of any trust-based relationship.
 Specific use cases will define the minimum types of identities required by a particular Relying Party as it evaluates Attestation Results, and perhaps additional associated Evidence.
 At a bare minimum, a Relying Party MUST start with the ability to verify the identity of a Verifier it chooses to trust.
-Attester identities may then be acquired through signed communications with the Verifier identity and/or the pre-provisioning Attester public keys in the Attester.
+Attester identities may then be acquired through signed or encrypted communications with the Verifier identity and/or the pre-provisioning Attester public keys in the Attester.
 
-During the Remote Attestation process, the Verifier's identity will be established with a Relying Party via a Verifier signature across recent Attestation Results.
+During the Remote Attestation process, the Verifier's identity must be established with a Relying Party, often via a Verifier signature across recent Attestation Results.
```
Comment (henkbirkholz) [inline draft-ietf-rats-ar4si.md:202]: Should we start capitalize MUSTs? There are several low cap musts in the text and this one caught my attention. It is mandatory that we check all occurrences if is not a "MUST" maybe we MUST use non-normative language that is not "must".

Comment (ericvoit) [reply draft-ietf-rats-ar4si.md:202]: I guess it depends what we consider normative from this document.   It certainly is a MUST from the perspective of the architecture document.   We likely should have a multi-person discussion on what to capitalize in this doc.

Comment (henkbirkholz) [reply draft-ietf-rats-ar4si.md:202]: Every must is a MUST and will be fixed to all caps by rfced (if that syntax reaches that gateway function). The multi-person discussion has to happen and we do not have to do that now in this PR.

**Review by henkbirkholz — APPROVED** [review]

**Pull request merged**